integration_test/testdata/setup_json_exporter.sh

```diff
@@ -0,0 +1,18 @@
+set -e
+
+wget -O json_exporter.tar.gz \
```

Comment:
Does this only work on certain distros? Should we exclude the others in the test?

Reply:
Right now this should work on all our linux distros. I was having trouble with tmux and screen since tmux is missing from SLES and screen is missing from Rocky. 
So in the test I try to install wget/python and either tmux or screen, and now it seems ok for all our linux distros.